composer.json

```diff
@@ -160,8 +160,11 @@
         "preferred-install": {
             "*": "dist"
         },
-        "sort-packages": true,
+        "sort-packages": true
+    },
+    "extra": {
         "branch-alias": {
+            "dev-master": "1.7-dev",
```

Comment:
We are not planning to release 1.7... Don't know too much about these aliases, but would it work to write `1.6-dev` here? :thinking:

Reply:
`1.6-dev` should be fine here but I'm not sure if it need to be on the master branch

Reply:
In my opinion the aliases should be something that comes 'next'. I mean master is ahead of current 1.6 right? When following semver it should be 1.7. Because you will stay at 1.6 I'll change it to 1.6-dev for you.